zapcli/zap_helper.py

```diff
@@ -394,5 +394,5 @@ def html_report(self, file_path):
     @staticmethod
     def _write_report(report, file_path):
         """Write report to the given file path."""
-        with open(file_path, 'w') as f:
+        with open(file_path, mode='wb') as f:
             f.write(report)
```

Comment:
Thanks for the change. Looks like this breaks in Python 3.5 with the error `TypeError: a bytes-like object is required, not 'str'`.

When you open in `b` mode in Python 3, you must pass bytes rather than a string (either works in Python 2). This should be fixable by changing the write line to:
```python
f.write(report.encode('utf-8'))
```

Let me know if that still works with your test page.